inc/admin/menus/class-sidebar.php

```diff
@@ -727,4 +727,13 @@ private function getContextSlug( string $page, bool $is_main_site_page ): string
 			( $is_main_site_page ? $page : network_admin_url( $page ) );
 	}
 
+	public static function handleH5pMenu(): void {
```

Comment:
Adding tests for this, should require the H5P plugin activation (or emulate it) in our tests.